includes/api/ApiBase.php

```diff
@@ -1568,4 +1579,38 @@ public static function debugPrint( $value, $name = 'unknown', $backtrace = false
 	public static function getBaseVersion() {
 		return __CLASS__ . ': $Id$';
 	}
+
+	// Wikia Change - BEGIN
+
+	/**
+	 * Sets up the WikiaLogger context for all log entries for current API call.
+	 * Adds extra information about module which handles the call and parameters
+	 * defined in mLogContextVariables array (if passed to a call).
+	 *
+	 * @param $params array Current query parameters
+	 */
+	protected function setupLogContext($params) {
+		$logContext = [];
+
+		foreach ( $params as $paramName => $paramValue) {
```

Comment:
how about iterating over $this->mLogContextVariables and checking if they are defined using eg. array_key_exists() ?